From 88271dc3231d06f2b53b0f15305f98f5288cfe06 Mon Sep 17 00:00:00 2001
From: "Ambrozewicz, Adrian" <adrian.ambrozewicz@intel.com>
Date: Fri, 30 Jul 2021 16:35:44 +0200
Subject: [PATCH] Add 'count' sensor type

PMT exposes data mainly in raw counter formats. This change makes
bmcweb aware of new sensor type.

Testing:
Values of type 'count' from PMT exposed successfully on Redfish

Signed-off-by: Alicja Rybak <alicja.rybak@intel.com>
---
 redfish-core/lib/sensors.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/redfish-core/lib/sensors.hpp b/redfish-core/lib/sensors.hpp
index 51c1d2ec..c547eaac 100644
--- a/redfish-core/lib/sensors.hpp
+++ b/redfish-core/lib/sensors.hpp
@@ -61,6 +61,7 @@ static auto powerPaths = std::to_array<std::string_view>({
 static auto sensorPaths = std::to_array<std::string_view>({
     "/xyz/openbmc_project/sensors/power",
     "/xyz/openbmc_project/sensors/current",
+    "/xyz/openbmc_project/sensors/count",
     "/xyz/openbmc_project/sensors/airflow",
     "/xyz/openbmc_project/sensors/humidity",
 #ifdef BMCWEB_NEW_POWERSUBSYSTEM_THERMALSUBSYSTEM
-- 
2.25.1

